.github/workflows/test-integration-rosa.yaml

```diff
@@ -0,0 +1,352 @@
+          fi
+        
+      - name: Launch tests
+        uses: ./.github/actions/chart-test-recipes
+        with:
+          platform: "${{ matrix.distro.platform }}"
+          type: "${{ matrix.distro.type }}"
+
+          identifier: "${{ env.INPUTS_IDENTIFIER }}"
+          persistent: "${{ env.TEST_PERSISTENT }}"
+          flow: "${{ matrix.flow }}"
+          test-enabled: "${{ env.INPUTS_TEST_ENABLED }}"
+
+          caller-git-ref: ${{ env.INPUTS_CALLER_GIT_REF }}
+          extra-values: ${{ env.INPUTS_EXTRA_VALUES }}
+          ingress-hostname-base: ${{ env.TEST_HOSTNAME_BASE }}
+
+          gh_app_id: ${{ secrets.GH_APP_ID_DISTRO_CI_MANAGE_GH_ENVS }}
+          gh_app_private_key: ${{ secrets.GH_APP_PRIVATE_KEY_DISTRO_CI_MANAGE_GH_ENVS }}
+
+  cleanup_clusters:
+    name: "Cleanup ROSA clusters"
+    if: always()
+    runs-on: ubuntu-latest
+    needs:
+      - define_matrix
+      - read_cluster_vars
+      - launch_tests
+    strategy:
+      fail-fast: false
+      matrix: 
+        distro: ${{ fromJson(needs.define_matrix.outputs.platform_matrix) }}
+    steps:
+      - name: Authenticate to AWS
+        run: |
+          aws configure set aws_secret_access_key ${{ secrets.DISTRO_CI_AWS_SECRET_KEY }} --profile=${{ secrets.DISTRO_CI_AWS_PROFILE }}
+          aws configure set region ${{ env.TEST_AWS_REGION }} --profile=${{ secrets.DISTRO_CI_AWS_PROFILE }}
+          aws configure set aws_access_key_id ${{ secrets.DISTRO_CI_AWS_ACCESS_KEY }} --profile=${{ secrets.DISTRO_CI_AWS_PROFILE }}
+
+      - name: Delete on-demand ROSA HCP Cluster
+        uses: camunda/camunda-tf-rosa/.github/actions/rosa-delete-cluster@main
+        if: always() && env.TEST_PERSISTENT != 'true'
+        timeout-minutes: 125
+        env:
+          AWS_PROFILE: ${{ secrets.DISTRO_CI_AWS_PROFILE }}
+        with:
+          rh-token: ${{ secrets.DISTRO_CI_REDHAT_CONSOLE_TOKEN }}
+          s3-backend-bucket: "${{ env.TF_S3_BUCKET }}-${{ env.TEST_AWS_REGION }}"
+
+          cluster-name: ${{ fromJson(needs.read_cluster_vars.outputs.result).cluster_name[matrix.distro.name] }}
+          aws-region: ${{ env.TEST_AWS_REGION }}
+  
+  report:
+    name: "Report failures"
+    if: always()
+    runs-on: ubuntu-latest
+    needs:
+      - launch_tests
+      - cleanup_clusters
+    steps:
+      - name: Create issue on failure
+        if: failure() && (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch')
+        uses: JasonEtco/create-an-issue@1b14a70e4d8dc185e5cc76d3bec9eab20257b2c5 # v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          WORKFLOW_URL: "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+        with:
+          assignees: "${{ env.ASSIGNEES_ISSUE_ERROR }}"
+          update_existing: true
+          search_existing: open
+          filename: .github/templates/issue_test_integration_rosa_failed.md
```

Comment:
again, would double check with the distribution team on how they handle workflow failures as this could get noisy.

Reply:
As mentioned above, let's use Slack for that.